tests/compile/test_fusion_attn.py

```diff
@@ -369,8 +258,13 @@ def test_attention_quant_pattern(num_qo_heads: int, num_kv_heads: int,
                     "in PyTorch 2.9+")
 
     monkeypatch.setenv("VLLM_USE_V1", "1")
-    if split_attention:
-        monkeypatch.setenv("VLLM_V1_USE_PREFILL_DECODE_ATTENTION", "1")
+
+    # Mock the _cached_get_attn_backend function to avoid cached results
```

Comment:
See #24604: we should fix `global_force_attn_backend_context_manager` instead of monkeypatching